### PR TITLE
docs(recipes): add publicPath + external/static html file

### PR DESCRIPTION
**Pull request by PlayMa256**
This PR contains:

- [ ] bugfix
- [ ] feature
- [ ] refactor
- [ ] tests
- [X] documentation
- [ ] metadata

### Breaking Changes?

- [ ] yes
- [X] no

If yes, please describe the breakage.

### Please Describe Your Changes

Add recipe to use an external html (not emmited by webpack) + public path.

### CODE REVIEW

File: recipes/external-html.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <main></main>
+    <script src="/assets/bundle.js"></script>
+  </body>
+</html>
+```
+
+You `webpack` config should be:
+
+```js
+const {resolve} = require('fs');
```
Comment (smashercosmo) [inline recipes/external-html.md]: I guess it should be 'path' instead of 'fs'

Comment (PlayMa256) [reply recipes/external-html.md]: Typo!! Good catch.

**Comment by EvHaus** [comment]
Thanks for this @PlayMa256  :)

**Comment by shellscape** [comment]
@PlayMa256 please review my changes

**Pull request merged**